sale_stock_picking_note/models/sale_order.py

```diff
@@ -25,7 +25,7 @@ class SaleOrder(models.Model):
     @api.depends("partner_id", "partner_shipping_id")
     def _compute_picking_notes(self):
         for order in self:
-            if order.state not in {"sale", "done", "cancel"}:
+            if order.state not in {"sale", "cancel"}:
```

Comment:
Shouldn't we take into account the `locked` field ?

Reply:
IMO it does not change the condition, i.e., if it is `locked` it will already be in the `sale` state.